catalog/pipeline/fbc-release/0.7/fbc-release.yaml

```diff
@@ -0,0 +1,113 @@
+      default: "[]"
+      description: List of additional tags the internal index image copy should be tagged with
+    - name: addArches
+      type: string
+      default: "[]"
+      description: List arches to be added to be built
+    - name: signingUser
+      type: string
+      description: User requesting to sign the FBC Index
+    - name: signingConfigMapName
+      type: string
+      description: The ConfigMap to be used by the signing Pipeline
+    - name: requestUpdateTimeout
+      type: string
+      description: Max seconds to wait until the status is updated
+    - name: buildTimeoutSeconds
+      type: string
+      description: Max seconds to wait until the build finishes
+  results:
+    - name: requestMessage
+      value: $(tasks.create-internal-request.results.requestMessage)
+    - name: requestReason
+      value: $(tasks.create-internal-request.results.requestReason)
+    - name: requestResults
+      value: $(tasks.create-internal-request.results.requestResults)
+  tasks:
+    - name: verify-enterprise-contract
+      taskRef:
+        name: verify-enterprise-contract
+        bundle: quay.io/hacbs-contract/ec-task-bundle:snapshot
+      params:
+        - name: IMAGES
+          value: $(params.snapshot)
+        - name: SSL_CERT_DIR
+          value: /var/run/secrets/kubernetes.io/serviceaccount
+        - name: POLICY_CONFIGURATION
+          value: $(params.enterpriseContractPolicy)
+        - name: STRICT
+          value: "1"
+    - name: create-internal-request
+      taskRef:
+        name: create-internal-request
+        bundle: quay.io/hacbs-release/task-create-internal-request:0.4
+      params:
+        - name: pipelineRunName
+          value: $(context.pipelineRun.name)
+        - name: fbcFragment
+          value: $(params.snapshot)
+        - name: fromIndex
+          value: $(params.fromIndex)
+        - name: overwriteFromIndex
+          value: $(params.overwriteFromIndex)
+        - name: binaryImage
+          value: $(params.binaryImage)
+        - name: buildTags
+          value: $(params.buildTags)
+        - name: addArches
+          value: $(params.addArches)
+        - name: requestUpdateTimeout
+          value: $(params.requestUpdateTimeout)
+        - name: buildTimeoutSeconds
+          value: $(params.buildTimeoutSeconds)
+      runAfter:
+        - verify-enterprise-contract
+    - name: sign-index-image
+      taskRef:
+        name: sign-index-image
+        bundle: quay.io/hacbs-release/task-sign-index-image:0.1
+      params:
+        - name: pipelineRunName
+          value: $(context.pipelineRun.name)
+        - name: requestJsonResults
+          value: $(tasks.create-internal-request.results.requestResults)
+        - name: signingUser
+          value: $(params.signingUser)
+        - name: configMapName
+          value: $(params.signingConfigMapName)
+        - name: requestUpdateTimeout
+          value: $(params.requestUpdateTimeout)
+      runAfter:
```

Comment:
you don't need this. Because a parameter is using a result from `create-internal-request`, tekton already knows it needs to run after

Reply:
ok. removed.